app/products/calls/actions/calls.ts

```diff
@@ -291,6 +292,12 @@ export const unraiseHand = () => {
     }
 };
 
+export const sendReaction = (emoji: CallReactionEmoji) => {
+    if (connection) {
+        connection.sendReaction(emoji);
```

Comment:
is this available with all servers that have calls enabled?

Reply:
see below